config/install.yaml

```diff
@@ -18329,6 +18329,8 @@ data:
             storage: 0
             replicas: 3
             duplicates: 60s
+            # 0: DiscardOld, 1: DiscardNew
+            discardPolicy: 0
```

Comment:
just curious, why do we have multiple configs where we have this field set?

Reply:
We use the different installation strategies for the numaflow controller, so replicating it for all.